tests/load/setup_k8s.sh

```diff
@@ -10,7 +10,7 @@ CLUSTER='autopush-locust-load-test'
 TARGET='https://updates-autopush.stage.mozaws.net'
 SCOPE='https://www.googleapis.com/auth/cloud-platform'
 REGION='us-central1'
-WORKER_COUNT=300
```

Comment:
The WORKER_COUNT found here has to correspond with the constant of the same name in [load.py](https://github.com/mozilla-services/autopush-rs/blob/master/tests/load/locustfiles/load.py#L47).